driver/ppm_flag_helpers.h

```diff
@@ -18,6 +18,7 @@ or GPL2.txt for full copies of the license.
 #endif
 
 #if !defined(UDIG) && !defined(__USE_VMLINUX__)
+#include <uapi/linux/memfd.h>
```

Comment:
I would keep the inlcude path similar to the other just for consistency so `<linux/memfd.h>`

Reply:
@Andreagit97 I tried doing that, but weirdly the macros in `memfd_create_flags_to_scap` don't execute. The flags don't propagate resulting in the value of the flag being zero in the tests.  I changed it to `<linux/memfd.h>` to ensure it's my local setup's problem.  If the `CI` fails, I'll change it back to `<uapi/linux/memfd.h>` 😄

Reply:
The problem persists:   https://app.circleci.com/pipelines/github/falcosecurity/libs/1230/workflows/f408e0fd-8a7b-4b64-9df3-f1e441d44650/jobs/3416?invite=true#step-106-188